server/test/services/applications/ProgramAdminApplicationServiceTest.java

```diff
@@ -107,12 +110,49 @@ public void getApplication() {
     assertThat(result.get().id).isEqualTo(application.id);
   }
 
+  @Test
+  public void getApplications() {
+    ProgramModel program = ProgramBuilder.newActiveProgram("test name", "test description").build();
+
+    ImmutableList<Long> appIdList = createApplicationList(3, program);
+
+    var result = service.getApplications(appIdList, program.getProgramDefinition());
+    assertThat(result).isNotEmpty();
+    assertThat(result.size()).isEqualTo(3);
+
+    result.stream()
+        .forEach(
+            e -> {
+              assertThat(appIdList.contains(e));
```

Comment:
```suggestion
              assertThat(appIdList).contains(e);
```
(blocking) This isn't actually asserting anything. I think this is what you want.